bxsolana/provider/grpc.ts

```diff
@@ -61,22 +61,35 @@ import { BaseProvider } from "./base.js"
 import { Client } from "@grpc/grpc-js"
 import { CallMetadataOptions } from "@grpc/grpc-js/src/call-credentials"
 import config from "../../utils/config.js"
+import { WsProvider } from "./ws"
 
 export class GrpcProvider extends BaseProvider {
     private client: Service
     private grpcClient: Client
 
-    constructor(address = `${MAINNET_API_GRPC_HOST}:${MAINNET_API_GRPC_PORT}`) {
+    constructor(address = `${MAINNET_API_GRPC_HOST}:${MAINNET_API_GRPC_PORT}`, useTLS: boolean) {
```

Comment:
casing: `useTls` in typescript. Typescript by convention doesn't capitalize acronyms